validator/db/kv/attester_protection.go

```diff
@@ -26,6 +27,37 @@ type AttestationRecord struct {
 	SigningRoot [32]byte
 }
 
+func NewPendingAttestationRecords() *PendingAttestationRecords {
```

Comment:
This conflicts with PendingAttestation used as part of prysm's core package. Throughout our code base, they refer to attestations that are queued to enter the chain, but perhaps we could choose a different name

Reply:
It's actually an official eth2 object https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/beacon-chain.md#pendingattestation

Reply:
Alright, I will rename